tests/test_main.py

```diff
@@ -501,7 +504,8 @@ def check_filename(self, filename, check_names):
                     continue
 
                 if isinstance(getattr(node, 'ctx', None), (ast.Store, ast.Del, getattr(ast, 'Param', ()))):
-                    assert not values
+                    # why is are there values ?
```

Comment:
What are you asking here? There aren't meant to be values before 3.11 because only actual expressions with values could be identified, not things being assigned or deleted. And there shouldn't be values right now either since opnames like `STORE_ATTR` are skipped in `check_code`.

Reply:
You are right. I figured out that my problem was that I map to many opcodes to ast-nodes. I'm currently extending the test cases. 

But that might take a while because I have currently something else to do.